### ISSUE TITLE
'Status' is an item, not a string

### ISSUE THREAD

**Issue by mzeinstra** [weso/SAF-Lux#33]
I see that 'Status' is mapped to a string, but is has controlled list. Should status not be item?

<img width="952" alt="Screenshot 2021-07-22 at 09 37 07" src="https://user-images.githubusercontent.com/5627431/126604452-ede1f85d-cb10-4a44-8af6-2d43edba932d.png">

**Comment by andrawaag** [comment]
@mzeinstra Good catch. This needs to be fixed in the DM_SAF rules. It is defined as a string but from a controlled list. The controlled list mentioned in CL8 is a list of internal identifiers.

**Comment by digitaalwerktuig** [comment]
@andrawaag Please refer to CL5 (not CL8). Can you add this to your property creator?
